src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -406,7 +406,7 @@ void testGetSchemas() throws SQLException {
     @Override
     void testGetFunctions() throws SQLException {
         // All function(s)
-        testGetFunctionsHelper("%", 19);
+        testGetFunctionsHelper("%", 15);
         // All function(s) with a 'S'
         testGetFunctionsHelper("%S%", 9);
```

Comment:
```suggestion
        testGetFunctionsHelper("%S%", 6);
```
Small update to the functions that have an `S`